lib/sda/service_test.go

```diff
@@ -444,6 +449,34 @@ func TestServiceBackForthProtocol(t *testing.T) {
 	assert.Equal(t, resp.Val, 10)
 }
 
+func TestClient_Send(t *testing.T) {
```

Comment:
It's IntelliJs doing. The case here is that `Client` is a structure and `Send` is a method on that structure. It follows https://golang.org/pkg/testing/#pkg-overview